<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>好词快记2.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="word_lineEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="word_tableWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string>单词</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>词性</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>释义</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>例句</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>音频</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>单词组</string>
    </property>
    <addaction name="group_new_action"/>
    <addaction name="group_delect_action"/>
    <addaction name="group_rename_action"/>
    <addaction name="separator"/>
    <addaction name="group_import_action"/>
    <addaction name="group_import_old_action"/>
    <addaction name="group_export_action"/>
    <addaction name="separator"/>
    <addaction name="group_update_data_action"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="help_about_action"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>单词</string>
    </property>
    <widget class="QMenu" name="word_add_menu">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>添加</string>
     </property>
     <addaction name="word_add_auto_word_action"/>
     <addaction name="word_add_inauto_word_action"/>
     <addaction name="word_add_batch_action"/>
    </widget>
    <addaction name="word_add_menu"/>
    <addaction name="word_delect_action"/>
    <addaction name="word_edit_action"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>学习</string>
    </property>
    <addaction name="study_meaning_action"/>
    <addaction name="study_spell_action"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>导出</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="setting_is_audio_need_download_action"/>
   </widget>
   <addaction name="menu_3"/>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="group_lineEdit"/>
     </item>
     <item>
      <widget class="QListWidget" name="group_listWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="group_new_action">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="group_delect_action">
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="group_rename_action">
   <property name="text">
    <string>重命名</string>
   </property>
  </action>
  <action name="group_import_action">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="group_import_old_action">
   <property name="text">
    <string>导入旧版</string>
   </property>
  </action>
  <action name="group_export_action">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="help_about_action">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="word_delect_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="word_update_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>更新数据</string>
   </property>
  </action>
  <action name="action_12">
   <property name="text">
    <string>打开网站</string>
   </property>
  </action>
  <action name="word_edit_action">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>编辑数据</string>
   </property>
  </action>
  <action name="word_add_auto_word_action">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>自动搜索</string>
   </property>
  </action>
  <action name="word_add_inauto_word_action">
   <property name="text">
    <string>手动输入</string>
   </property>
  </action>
  <action name="group_update_data_action">
   <property name="text">
    <string>更新数据</string>
   </property>
  </action>
  <action name="word_add_batch_action">
   <property name="text">
    <string>批量添加</string>
   </property>
  </action>
  <action name="study_meaning_action">
   <property name="text">
    <string>背释义</string>
   </property>
  </action>
  <action name="study_spell_action">
   <property name="text">
    <string>背拼写</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>详细信息</string>
   </property>
  </action>
  <action name="setting_is_audio_need_download_action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>缓存音频数据</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
